registration/tests/enhanced_backend.py

```diff
@@ -0,0 +1,258 @@
+import datetime
```

Comment:
Seeing the code once more, I think perhaps I could follow the same logic with the class variable on probably most of the views in `registration/backends/admin_approval/views.py`. That is, altering the views on `registration/backends/default/views.py` to have a class variable and store the registration profile class and sublass them altering this value. I think this would be cleaner.